src/views/empty-state.html

```diff
@@ -2,13 +2,13 @@
   <div ng-if="$ctrl.config.icon" class="blank-slate-pf-icon">
     <span class="{{$ctrl.config.icon}}"></span>
   </div>
-  <h1 id="title">
+  <h4 id="empty-state-title-{{$id}}" class="h1 title">
```

Comment:
Can we avoid using title as a class and just use this new id as a selector in the tests?